src/SWP/Component/Archiver/Archiver/ZipArchiver.php

```diff
@@ -0,0 +1,44 @@
+<?php
```

Comment:
We should use it also in `OrganizationAwareThemeUploader`. It requires a little more functionality, but at the end does this same.

Reply:
another PR